pkg/health/health.go

```diff
@@ -103,6 +113,14 @@ func findPortByName(container api.Container, portName string) int {
 	return -1
 }
 
+func timeout(container api.Container) time.Duration {
```

Comment:
The default value cannot be initialized in this way. The value should be stated in the field descriptions in the versioned types.go files, and should be populated in the field such that if the user does a GET on the object, they'll see the value, as discussed in #1502 and #2587. Pending the "right" solution, the value can be initialized in pkg/api/validation/validation.go, as in #3587.